rdrf/registry/common/middleware.py

```diff
@@ -46,8 +46,6 @@ class UserSentryMiddleware(MiddlewareMixin):
     """
 
     whitelisted_views = [
-        "login",
```

Comment:
This breaks "Require two-factor authentication"

Reply:
Added back in. As well as added in `favicon` as it was also being requested.